xblock/run_script.py

```diff
@@ -13,6 +14,6 @@ def run_script(pycode):
 
     # execute it.
     globs = {}
-    exec pycode in globs, globs  # pylint: disable=W0122
+    exec(pycode, globs, globs)  # pylint: disable=W0122
```

Comment:
Won't this break python2?  I'm pretty sure I had to do a `six.exec_()` in my port to work around exec incompatibilities.